packages/cli/src/commands/alias/ls.ts

```diff
@@ -22,13 +23,23 @@ export default async function ls(
 ) {
   const { output } = client;
   const { '--next': nextTimestamp } = opts;
+  const { '--limit': limit } = opts;
   const { contextName } = await getScope(client);
 
   if (typeof nextTimestamp !== undefined && Number.isNaN(nextTimestamp)) {
     output.error('Please provide a number for flag --next');
     return 1;
   }
 
+  if (
+    typeof nextTimestamp !== undefined &&
```

Comment:
**suggestion (blocking):** Remove the check for `nextTimestamp`

The current condition `typeof nextTimestamp !== undefined` will always be true because if `nextTimestamp` is undefined, `typeof nextTimestamp` is the string `"undefined"`, not the value `undefined`.

I don't think we want to require `--next` for this `--limit` validation to show an error. So, I think we should remove this check entirely from each occurrence in this PR.

Reply:
Oops as you pointed out, `nextTimestamp` shouldn't even be in this check because this is a check for limit. Sorry about that and thanks for spotting.